### PR TITLE
Enable CA1805 diagnostics

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions/src/MvcViewDocumentClassifierPass.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public MvcViewDocumentClassifierPass(bool useConsolidatedMvcViews)
             {
                 @class.ClassName = className;
             }
-            
+
```
Comment (TanayParikh) [inline src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions/src/MvcViewDocumentClassifierPass.cs:56]:
```suggestion

```

**Review by TanayParikh — APPROVED** [review]
LGTM 👍

File: src/Features/JsonPatch/src/Internal/ParsedPath.cs [hunk 2]
```diff
@@ -14,8 +14,6 @@ namespace Microsoft.AspNetCore.JsonPatch.Internal
     /// </summary>
     public readonly struct ParsedPath
     {
-        private static readonly string[] Empty = null;
```
Comment (Tratcher) [inline src/Features/JsonPatch/src/Internal/ParsedPath.cs:17]: This smells like a bug, are we sure this wasn't supposed to return `Array.Empty<byte>()`?

**Review by Tratcher — APPROVED** [review]

File: src/Features/JsonPatch/src/Internal/ParsedPath.cs (hunk 2 above)
Comment (pranavkm) [reply src/Features/JsonPatch/src/Internal/ParsedPath.cs:17]: Probably, but I don't really feel like digging up JsonPatch code 😄

**Pull request merged**